metadata.rb

```diff
@@ -4,7 +4,9 @@
 license           "Apache 2.0"
 description       "Installs/Configures drbd."
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           "0.9.1"
+version           "0.9.2"
```

Comment:
In general don't bump the versions in PRs. We have tooling that handles this automatically